### PR TITLE
Feature/wms 886 extrinsic view optimization

### PR DESCRIPTION
**Pull request by chernetsky**
Part of #IWMS-886

Extrinsics amount and fee fields processing.

### CODE REVIEW

File: lib/extrinsicData.js [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
+  }
+
+  function getMethod() {
+    const { method } = extrinsic.toHuman();
+    const result = {};
+    result.section = method.section;
+    result.method = method.method;
+    return result;
+  }
+
```
[the remark this answers is not in the document]
Comment (chernetsky) [reply lib/extrinsicData.js:22]: Ok!

File: constants.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export const EVENT_PHASE = {
```
Comment (sswebcoder) [inline constants.ts:1]: Maybe better use enum?

File: crawlers/blockListener.ts [hunk 3]
```diff
@@ -105,10 +130,12 @@ export class BlockListener {
         });
       }
     }
+
+    return parsedEvents;
   }
 
-  async getBlockData(blockNumber: number) {
-    return blockData.get({
+  async _getBlockData(blockNumber: number) {
```
Comment (sswebcoder) [inline crawlers/blockListener.ts:137]: We should use TS accessor methods.

File: crawlers/blockListener.ts [hunk 4]
```diff
@@ -118,4 +145,11 @@ export class BlockListener {
 export async function start({ api, sequelize }: ICrawlerModuleConstructorArgs) {
   const blockListener = new BlockListener(api, sequelize);
   await blockListener.startBlockListening();
+
+  // todo: debug purpose
```
Comment (sswebcoder) [inline crawlers/blockListener.ts:149]: DBG :)

File: crawlers/blockListener.ts (hunk 3 above)
Comment (chernetsky) [reply crawlers/blockListener.ts:137]: Done

File: constants.ts (hunk 2 above)
Comment (chernetsky) [reply constants.ts:1]: Done

**Pull request merged**